app/src/main/java/com/github/se/icebreakrr/model/profile/ProfilesViewModel.kt

```diff
@@ -139,10 +144,10 @@ open class ProfilesViewModel(
   /** Initializes the repository and fetches profiles. */
   init {
     repository.init {
-      // Fetch profiles on initialization
+      val defaultPoint = GeoPoint(DEFAULT_USER_LATITUDE, DEFAULT_USER_LONGITUDE)
       getSelfProfile {
-        getFilteredProfilesInRadius(
-            GeoPoint(DEFAULT_USER_LATITUDE, DEFAULT_USER_LONGITUDE), DEFAULT_RADIUS)
+        getFilteredProfilesInRadius(defaultPoint, DEFAULT_RADIUS)
+        getMessagingRadiusProfile(defaultPoint)
```

Comment:
Why do you call `getMessagingRadiusProfile` with the default location ?

Reply:
It was what Arthur was doing before and it worked well, i just refactor his code to make it clearer

Reply:
No, before arthur called `getFilteredProfilesInRadius` with the default location, not `getMessagingRadiusProfile` if github shows me well the diffs. 
Edit: if it works I guess it is okay.